src/fastener.js

```diff
@@ -182,3 +193,10 @@ const everywhereG = f => z =>
 const everywhereU = (f, z) =>
   modifyU(f, transformMoveU(downHead, everywhereG(f), z))
 export const everywhere = curry2(everywhereU)
+
+function pathOfA(ks, z) {
+  const k = keyOf(z)
+  return isDefined(k) ? pathOfA([k, ...ks], up(z)) : ks
+}
+export const pathOf = z => pathOfA([], z)
```

Comment:
This works, but can be a bit expensive.  It should be possible to create a version that is O(n) in the length of the path by exploiting the zipper data structure and follow the `up` chain.

Consider the data structure formed by the following example:

```js
seq([[[[[[[[]]]]]]]],
    F.toZipper,
    F.downTo(0),
    F.downTo(0),
    F.downTo(0),
    F.downTo(0),
    F.downTo(0),
    F.downTo(0),
    F.downTo(0))
```

The path can be extracted from the `up*.key` elements in O(n) time.